media/whip_connection.go

```diff
@@ -212,9 +226,16 @@ func (m *MediaState) Stats() (*MediaStats, error) {
 		if s == nil {
 			continue
 		}
+		jitter := s.InboundRTPStreamStats.Jitter
+		switch t.Kind() {
+		case webrtc.RTPCodecTypeAudio:
+			jitter = jitter / audioTimeBase
+		case webrtc.RTPCodecTypeVideo:
+			jitter = jitter / videoTimeBase
+		}
 		trackStats = append(trackStats, TrackStats{
-			Kind:            t.Kind(),
-			Jitter:          s.InboundRTPStreamStats.Jitter,
+			Type:            TrackType{t.Kind()},
+			Jitter:          jitter,
```

Comment:
The jitter calculation can probably be simplified to something like

```suggestion
			Jitter:          s.InboundRTPStreamStats.Jitter / t.Codec().ClockRate
```

Reply:
ah awesome yep that works thanks